### PR TITLE
CheckBox: add a Getting Started tutorial

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/10 Specify the Initial Value.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+The CheckBox can be in one of the following states depending on the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property's value:
```
Comment (khamlyuk) [inline concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/10 Specify the Initial Value.md]:
```suggestion
The CheckBox can be in one of the following states depending on its [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value):
```

File: concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/10 Specify the Initial Value.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+- checked  
+[value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `true`
```
Comment (khamlyuk) [inline concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/10 Specify the Initial Value.md]:
```suggestion
the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `true`
```

File: concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/10 Specify the Initial Value.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+- unchecked  
+[value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `false`
```
Comment (khamlyuk) [inline concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/10 Specify the Initial Value.md]:
```suggestion
the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `false`
```

File: concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/10 Specify the Initial Value.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+- indeterminate 
+[value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `undefined` or `null`
```
Comment (khamlyuk) [inline concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/10 Specify the Initial Value.md]:
```suggestion
the [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) is `undefined` or `null`
```

File: concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/10 Specify the Initial Value.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+The following code specifies the initial value as `null`.
```
Comment (khamlyuk) [inline concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/10 Specify the Initial Value.md]:
```suggestion
The following code specifies the initial value as `null`:
```

File: concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/15 Configure the CheckBox.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
+To make the CheckBox more accessible, specify the [text](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#text) and [hint](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#hint) properties.
```
Comment (khamlyuk) [inline concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/15 Configure the CheckBox.md]: Something is off here. Can you clarify?

File: concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/20 Handle the Value Change.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
+Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) handler to perform an action when users click the CheckBox.
```
Comment (khamlyuk) [inline concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/20 Handle the Value Change.md]:
```suggestion
Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) event handler to perform an action when users click the CheckBox.
```

**Review by khamlyuk — APPROVED** [review]

File: concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/15 Configure the CheckBox.md (hunk 6 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/15 Configure the CheckBox.md]: > To hint users the CheckBox purpose, specify the [text](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#text) and [hint](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#hint) properties.

or simply

> To label the CheckBox and add a pop-up hint, specify the [text](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#text) and [hint](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#hint) properties.

Comment (khamlyuk) [reply concepts/05 UI Components/CheckBox/00 Getting Started with CheckBox/15 Configure the CheckBox.md]: I prefer the second one.

File: concepts/Getting Started/Widget Basics - AngularJS/05 Change Options.md [hunk 8]
```diff
@@ -14,7 +14,7 @@ All operations with UI component properties are carried out through the scope pr
         $scope.checkBoxValue = true;
     }
```
Comment (albertov05) [inline concepts/Getting Started/Widget Basics - AngularJS/05 Change Options.md:16]: Please update line 1 (above) as follows:

`All operations with UI component properties are carried out through the scope properties to which these properties are bound. To bind a UI component property to a scope property, use the **bindingOptions** object as shown in the following code. Note that the scope property name in this object is enclosed in quotes.`

File: concepts/Getting Started/Widget Basics - AngularJS/05 Change Options.md [hunk 9]
```diff
@@ -14,7 +14,7 @@ All operations with UI component properties are carried out through the scope pr
...
 
-Now, if you change the `checkBoxValue` scope property in code, the [CheckBox](/concepts/05%20UI%20Components/CheckBox/00%20Overview.md '/Documentation/Guide/UI_Components/CheckBox/Overview/') will receive the changes. And vice versa, if an end user selects the CheckBox in the UI, the `checkBoxValue` scope property will be updated.
+Now, if you change the `checkBoxValue` scope property in code, the [CheckBox](/Documentation/Guide/UI_Components/CheckBox/Getting_Started_with_CheckBox/) will receive the changes. And vice versa, if an end user selects the CheckBox in the UI, the `checkBoxValue` scope property will be updated.
 
 If you bind a UI component to a collection, the UI component gets updated only when an object is added to or removed from this collection. To make the UI component listen for changes even in the _fields_ of any object in the collection, configure the **bindingOptions** object as follows.
```
Comment (albertov05) [inline concepts/Getting Started/Widget Basics - AngularJS/05 Change Options.md:19]:
```suggestion
If you bind a UI component to a collection, the UI component gets updated only when an object is added to or removed from this collection. To make the UI component listen for changes even in the _fields_ of any object in the collection, configure the **bindingOptions** object as follows:
```

**Pull request merged**